lib/relay/client/static.go

```diff
@@ -173,7 +173,7 @@ func (c *staticClient) disconnect() {
 }
 
 func (c *staticClient) join() error {
-	if err := protocol.WriteMessage(c.conn, protocol.JoinRelayRequest{}); err != nil {
+	if err := protocol.WriteMessage(c.conn, protocol.JoinRelayRequest{Token: c.uri.Query().Get("token")}); err != nil {
```

Comment:
The token here should be parser from the URI when the staticClient is constructed, so that we don't need to re-parse it on every join.